vector/src/Data/Vector/Generic/Mutable.hs

```diff
@@ -673,32 +671,32 @@ clone v = do
 -- -------
 
 -- | Grow a vector by the given number of elements. The number must not be
--- negative otherwise error is thrown. Semantics of this function is exactly the
--- same as `unsafeGrow`, except that it will initialize the newly
+-- negative, otherwise an exception is thrown. The semantics of this function
+-- are exactly the same as of 'unsafeGrow', except that it will initialize the newly
```

Comment:
FYI Backtick and tick are equivalent in haddock when it comes to linking functions. So that change was unnecessary: 
```
`unsafeGrow` -> 'unsafeGrow'
```

However, there is no need to change it back. I am just mentioning it because I prefer to use backticks and I am sure I'll be adding more of those in the future.

Reply:
I'm aware that ticks and backticks are equivalent, but my impression was that ticks are generally preferred. I will keep that in mind though.

Reply:
Ticks was the original approach, backticks were added much much later. Neither is preferred over another. I like it because t is more consistent with markdown